### DISCUSSION TITLE
clickcounter - plugin that counts how often links have been clicked

### DISCUSSION THREAD

**Discussion by waschinski** [shaarli/Shaarli#2061] [Show and tell]
I made a plugin that counts the clicks of all external links and displays them in the `link_plugin` section: https://github.com/waschinski/shaarli-clickcounter

**Comment by nodiscc** [comment]
Hi @waschinski 
Same as https://github.com/shaarli/Shaarli/discussions/2062
 Would you like to create a Pull Request to add it to https://github.com/shaarli/Shaarli/blob/master/doc/md/Community-and-related-software.md#third-party-plugins ?

**Comment by nodiscc** [comment]
Closing as this is now listed on https://shaarli.readthedocs.io/en/master/Community-and-related-software.html#third-party-plugins
